.gitignore

```diff
@@ -0,0 +1,2 @@
+
+*.xml
```

Comment:
why do you add xml?

Reply:
This has been added so that extra files created by the PyCharm editor are ignored by Git.

Reply:
Please delete it, because xml is not unrelated module.